tests/test_lab/test_gates_fock.py

```diff
@@ -210,6 +217,43 @@ def test_raise_interferometer_error():
         RealInterferometer(num_modes=num_modes, modes=modes)
 
 
+@given(phase_stdev=medium_float.filter(lambda x: x > 0))
+def test_phasenoise_creates_dm(phase_stdev):
```

Comment:
Imho there's a discrepancy between the title of the test, the doc of the test and what the test actually evaluates. None of them is wrong but the fact that there's no match is a pity.
Testing that something isn't None doesn't tell you what it is, it tells it's not nothing. (what the test does)
Testing that something (a DM) is created doesn't entail it's correct, it merely guarantees it exists (name vs doc)